src/CommonITILCost.php

```diff
@@ -662,10 +662,6 @@ function viewAddCost{{ id }}_{{ rand }} (btn) {
             echo Html::scriptBlock(<<<JS
                 $(() => {
                     $('#datatable_costs{$ID}{$rand}').on('click', 'tbody tr', (e) => {
-                        //ignore click in first column (the massive action checkbox)
-                        if ($(e.target).closest('td').is('td:first-child')) {
-                            return;
-                        }
```

Comment:
This doesn't make sense as we explicitly set `'showmassiveactions' => false,` in the lines just above.